pages/add-clinic.vue

```diff
@@ -141,8 +136,24 @@ export default {
         alert("Please fill out all of the fields!");
       }
     },
-    validate() {
-      this.$refs.form.validate();
+    errorMessages(field, name) {
+      if (field.$dirty && field.$invalid) {
+        return field
+          .$flattenParams()
+          .filter((param) => !field[param.name])
+          .map((param) => {
+            let args = {};
```

Comment:
should explain what `args` these are. `translationPosition`?

(I don't actually know what the second param of `$t` does)

Reply:
Renamed it to `templateArgs` -- it's for plugging into variables into the template strings.